docs/usage.md

```diff
@@ -6,61 +6,143 @@
+
+## Supported analyses
+
+A variety of analyses are accessible in oncoanalyser and are implicitly run according to the data described in the
+samplesheet. The supported analysis types for each workflow are listed below.
+
+| Input sequence data                 |  WGS/WTS workflow  | Targeted sequencing workflow<sup>\*</sup> |
+| ----------------------------------- | :----------------: | :---------------------------------------: |
+| • Tumor/normal DNA<br />• Tumor RNA | :white_check_mark: |                     -                     |
+| • Tumor only DNA<br />• Tumor RNA   | :white_check_mark: |            :white_check_mark:             |
+| • Tumor/normal DNA                  | :white_check_mark: |                     -                     |
+| • Tumor only DNA                    | :white_check_mark: |            :white_check_mark:             |
+| • Tumor only RNA                    | :white_check_mark: |                     -                     |
+
+<sub><sup>\*</sup> Supported analyses relate to the TSO500 panel only</sub>
+
+## Samplesheet
+
+A samplesheet that contains information of each input in CSV format is needed to run oncoanalyser. The required input
+details and columns are [described below](#column-descriptions).
+
+Several different input filetypes beyond FASTQ and BAM are recognised, including intermediate output files generated
+during execution such as the PURPLE output directory. The full list of recognised input filetypes is available
+[here](https://github.com/nf-core/oncoanalyser/blob/0.4.5/lib/Constants.groovy#L58-L90).
+
+### Simple example
+
+#### FASTQ
+
+> [!NOTE]
+> Currently only non-interleaved paired-end reads are accepted as FASTQ input
+
+```csv title="samplesheet.csv"
+group_id,subject_id,sample_id,sample_type,sequence_type,filetype,info,filepath
+P1__wgts,P1,SA,normal,dna,fastq,library_id:SA_library;lane:001,/path/to/P1.SA.normal.dna.wgs.001.R1.fastq.gz;/path/to/P1.SA.normal.dna.wgs.001.R2.fastq.gz
+P1__wgts,P1,SB,tumor,dna,fastq,library_id:SB_library;lane:001,/path/to/P1.SB.tumor.dna.wgs.001.R1.fastq.gz;/path/to/P1.SB.tumor.dna.wgs.001.R2.fastq.gz
+P1__wgts,P1,SC,tumor,rna,fastq,library_id:SC_library;lane:001,/path/to/P1.SC.tumor.rna.wts.001.R1.fastq.gz;/path/to/P1.SC.tumor.rna.wts.001.R2.fastq.gz
 ```
 
-### Multiple runs of the same sample
+#### BAM
 
-The `sample` identifiers have to be the same when you have re-sequenced the same sample more than once e.g. to increase sequencing depth. The pipeline will concatenate the raw reads before performing any downstream analysis. Below is an example for the same sample sequenced across 3 lanes:
+> [!NOTE]
+> Inputs with the `bam` filetype will be processed by MarkDups as required by hmftools. Where an input BAM has already
+> been processed by MarkDups, you can avoid needless reprocessing by setting `bam_markdups` as the filetype instead.
```

Comment:
Change 'MarkDups' to 'HMF MarkDups' (if it's important that HMF MarkDups has been run and not some other duplicate marking tool)